modules/ROOT/pages/license-key.adoc

```diff
@@ -0,0 +1,83 @@
+= License key
+:description_short: {productname} License Configuration Guide | {productname}
+:description: Learn how to configure license key and stop {productname} from running in the evaluation mode.
+:keywords: {productname}, cloud, script, textarea, apiKey, faq, license key, frequently asked questions,
+
+[IMPORTANT]
+====
+{productname} 7 is licensed under the GNU General Public License Version 2 or later. A new configuration option called 'license_key' requires developers to make a conscious decision to use {productname} with the GPLv2+ license or with a commercial license.
+
+If you are using {productname} in a self-hosted environment, a console log warning message will display if the license key config option is missing or invalid. This message aims to ensure compliance with licensing requirements and provide transparency during the evaluation period.
+
+This message will not be shown when loading {productname} from the Tiny Cloud, as it is already under a commercial license.
+====
+
+== Setting the license
+
+=== Use TinyMCE with the GPLv2+ license:
+
+If the developer intends to self-host {productname} under the GPL license, they can set the 'license_key' config option to 'gpl'. Case sensitivity does not matter.
+
+==== Example
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  license_key: 'gpl'
+});
+----
+
+=== Use TinyMCE with a commercial license:
+
+If the developer intends to self-host {productname} under a commercial license, they must provide a valid license key. Customers who have purchased a self-hosted-eligible license for {productname} will find their license key in the account portal. To purchase a commercial license, see available options on the link:{pricingpage}/[pricing page]. 
+
+==== Example
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  license_key: '<your-license-key>'
+});
+----
+
+== FAQ
+
+=== What does the GPL license mean?
+
+TinyMCE 7 is licensed under the GNU General Public License Version 2 or later, often abbreviated to GPLv2+. Read more about the license terms here.
+
+The GPLv2+ license was chosen to provide the best compatibility with existing GPL licensed open source projects.
+
+=== What is the difference between a license key and the API key?
+
+The API key is used when loading TinyMCE from the Tiny Cloud. The license key is used to declare the license terms when self-hosting TinyMCE.
```

Comment:
emphasize (bold or italic) the terms "API key" and "license key" to emphasize the are discreet concepts